### ISSUE TITLE
_update.update_all(self, rs=None, since=None) fixed!

### ISSUE THREAD

**Issue by svaio** [peeringdb/peeringdb-py#34]
The rs value is None. There is a for loop with None -> throws exception. 

Fix attached.

~~~
Index: peeringdb/_update.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- peeringdb/_update.py	(revision 2142ccf40669c30a847736e1b4c5dabe0b113498)
+++ peeringdb/_update.py	(date 1575993356532)
@@ -57,10 +57,9 @@
 
     def update_all(self, rs=None, since=None):
         "Sync all objects for the relations rs (if None, sync all resources)"
-        self._log.info("Updating resources: %s", ' '.join(r.tag for r in rs))
-
         if rs is None:
             rs = resource.all_resources()
+        self._log.info("Updating resources: %s", ' '.join(r.tag for r in rs))
         ctx = self._ContextClass(self)
         for r in rs:
             self._atomic_update(lambda: ctx.sync_resource(r, since=since))
~~~

**Comment by arnoldnipper** [comment]
@vegu and/or @grizz, please have a look

**Comment by vegu** [comment]
This has been fixed in d275027d77e0db396e7c8d652e4efc033ff65d84
